.devcontainer/images/.claude/agents/devops-executor-linux.md

```
@@ -36,11 +35,54 @@ allowed-tools:
+    fallback: "Handle directly using generic Linux knowledge below"
+
+  dispatch_pattern: |
+    1. Read /etc/os-release (or context from caller)
+    2. Match ID to routing_table
+    3. IF match found:
+       Task(subagent_type=<agent_name>, prompt="<original_query>")
+    4. ELSE: Handle directly with generic knowledge below
+```
+
+**Example dispatch:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)